<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormDeviceDetail</class>
 <widget class="QWidget" name="FormDeviceDetail">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>test</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="3" column="2">
       <widget class="QPushButton" name="pushButton_set_config">
        <property name="text">
         <string>set_config</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="pushButton_connect">
        <property name="text">
         <string>connect</string>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QPushButton" name="pushButton_del_cam">
        <property name="text">
         <string>del_cam</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="3">
       <widget class="QTreeWidget" name="treeWidget_device">
        <property name="contextMenuPolicy">
         <enum>Qt::CustomContextMenu</enum>
        </property>
        <column>
         <property name="text">
          <string notr="true">1</string>
         </property>
        </column>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLineEdit" name="lineEdit_connect">
        <property name="text">
         <string>192.168.1.216</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="pushButton_get_config">
        <property name="text">
         <string>get_config</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QPushButton" name="pushButton_add_cam">
        <property name="text">
         <string>add_cam</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="pushButton_search">
        <property name="text">
         <string>search</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QLineEdit" name="lineEdit_get_config"/>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QLineEdit" name="lineEdit_set_config"/>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="QLineEdit" name="lineEdit_add_cam"/>
      </item>
      <item row="5" column="0" colspan="2">
       <widget class="QLineEdit" name="lineEdit_del_cam"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
